src/integration/java/io/pinecone/integration/controlPlane/serverless/CreateDescribeListAndDeleteIndexTest.java

```diff
@@ -1,52 +1,131 @@
 package io.pinecone.integration.controlPlane.serverless;
 
 import io.pinecone.clients.Pinecone;
+import io.pinecone.exceptions.PineconeBadRequestException;
+import io.pinecone.exceptions.PineconeException;
+import io.pinecone.exceptions.PineconeNotFoundException;
+import io.pinecone.exceptions.PineconeUnmappedHttpException;
 import io.pinecone.helpers.RandomStringBuilder;
-import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 import org.openapitools.client.model.*;
 
 import static io.pinecone.helpers.IndexManager.waitUntilIndexIsReady;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.*;
 
 public class CreateDescribeListAndDeleteIndexTest {
-    private Pinecone controlPlaneClient;
-
-    @BeforeEach
-    public void setUp() {
-        controlPlaneClient = new Pinecone(System.getenv("PINECONE_API_KEY"));
-    }
-
-    @Test
-    public void createAndDelete() throws InterruptedException {
-        String indexName = RandomStringBuilder.build("index-name", 8);
+    private static Pinecone controlPlaneClient = new Pinecone(System.getenv("PINECONE_API_KEY"));
+    private static final String indexName = RandomStringBuilder.build("create-index", 8);
+    @BeforeAll
+    public static void setUp() throws InterruptedException {
+        // Create the index
         ServerlessSpec serverlessSpec = new ServerlessSpec().cloud(ServerlessSpec.CloudEnum.AWS).region("us-west-2");
```

Comment:
Might want to have a constant (or similar Java concept) for the `region` here in anticipation of us expanding to more regions?